JASP-Desktop/results/resultsjsinterface.cpp

```diff
@@ -73,6 +73,7 @@ void ResultsJsInterface::setResultsLoaded(bool resultsLoaded)
 		emit runJavaScript("window.setAppVersion('" + version + "')");
 
 		setGlobalJsValues();
+		setFontFamily(Settings::value(Settings::RESULT_FONT).toString());
```

Comment:
Just check here if "default is on" and if so don't set the result font especially.